src/Integration.Vsix/CFamily/ProcessRunner.cs

```diff
@@ -136,9 +141,22 @@ public bool Execute(ProcessRunnerArguments runnerArgs)
             return succeeded;
         }
 
-        #endregion Public methods
-
-        #region Private methods
+        private static Action KillProcess(Process process)
+        {
+            return () =>
+            {
+                try
+                {
+                    if (process != null && !process.HasExited)
+                    {
+                        process.Kill();
+                    }
+                }
+                catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
+                {
+                }
```

Comment:
Is there a reason not to log the error message?

Reply:
It's possible that the process has exited just between the IF check and before Kill() is called, in which case an exception is thrown. I think logging it will create un-necessary noise.